conf/application.conf

```diff
@@ -127,6 +127,12 @@ microservice {
       port = 8470
     }
 
+    advance-valuation-rulings-frontend {
+      protocol = http
+      host = localhost
+      port = 12600
+    }
```

Comment:
Would it create a cyclic dependency if the backend is also making requests to the frontend?